src/main/resources/application.properties

```diff
@@ -0,0 +1,9 @@
+spring.jpa.hibernate.ddl-auto=update
+
+spring.jpa.show-sql=true
+
+
+spring.datasource.url=jdbc:mysql://localhost:3306/nelp
+spring.datasource.username=root
+spring.datasource.password=csedbadmin
+spring.datasource.driverClassName=com.mysql.jdbc.Driver
```

Comment:
https://softwareengineering.stackexchange.com/questions/246412/database-password-problem-when-using-git-to-manage-project-that-uses-spring-to-i/246413